opencti-platform/opencti-front/src/private/components/data/csvMapper/CsvMapperCreation.tsx

```diff
@@ -9,15 +9,16 @@ import { CsvMapperAddInput } from '@components/data/csvMapper/__generated__/CsvM
 import { insertNode } from '../../../../utils/store';
 
 const csvMapperCreation = graphql`
-  mutation CsvMapperCreationContainerMutation($input: CsvMapperAddInput!) {
-    csvMapperAdd(input: $input) {
-      id
-      name
-      has_header
-      separator
-      errors
+    mutation CsvMapperCreationContainerMutation($input: CsvMapperAddInput!) {
```

Comment:
why the indent changes ?
We're supposed to be 2-space everywhere I think

Reply:
Yes, I put 2-space inside IDE and run the lint. Automatically get to this indentation. Any clue on how to resolve this on intelliji ?

Reply:
Well in fact I have the same issue as you apparently... There is some conflicting options maybe.
Anyone experienced with IntelliJ ? @Kedae maybe ?